herbieapp/controllers/save_business_entity_controller.py

```diff
@@ -13,15 +13,27 @@
 
 class SaveBusinessEntityController(APIView):
     _entity_manager = None
+    _logger = None
+    _validator = None
+    _schema_registry = None
+    _permission_manager = None
+    _permission_classes = None
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        self._entity_manager = BusinessEntityManager()
-        self._logger = logging.getLogger(__name__)
-        self._validator = JsonSchemaValidator()
-        self._schema_registry = SchemaRegistry()
+    def __init__(
+            self,
+            _entity_manager: BusinessEntityManager,
+            _logger,
+            _validator: JsonSchemaValidator,
+            _schema_registry: SchemaRegistry,
+            _permission_manager: PermissionManager
+    ):
+        #super().__init__(**kwargs)
```

Comment:
I something happening in the APIView class-constructor?

Reply:
yeah I should check this one, if this call does something